deno/rest/v10/monetization.ts

```diff
@@ -0,0 +1,85 @@
+import type { Snowflake } from '../../globals.ts';
+import type { APIEntitlement, APISKU } from '../../v10.ts';
+
+/**
+ * https://discord.com/developers/docs/monetization/entitlements#list-entitlements
+ */
+export interface RESTGetAPIEntitlementsQuery {
+	/**
+	 * User ID to look up entitlements for
+	 */
+	user_id?: Snowflake | undefined;
+	/**
+	 * Optional list of SKU IDs to check entitlements for
+	 * Comma-delimited set of snowflakes
+	 */
+	sku_ids?: string | undefined;
+	/**
+	 * Retrieve entitlements before this entitlement ID
+	 */
+	before?: Snowflake | undefined;
+	/**
+	 * Retrieve entitlements after this entitlement ID
+	 */
+	after?: Snowflake | undefined;
+	/**
+	 * Number of entitlements to return (1-100)
+	 *
+	 * @default 100
+	 */
+	limit?: number | undefined;
+	/**
+	 * Guild ID to look up entitlements for
+	 */
+	guild_id?: Snowflake | undefined;
+	/**
+	 * Whether ended entitlements should be omitted
+	 */
+	exclude_ended?: boolean | undefined;
+}
+
+/**
+ * https://discord.com/developers/docs/monetization/entitlements#list-entitlements
+ */
+export type RESTGetAPIEntitlementsResult = APIEntitlement[];
+
+/**
+ * https://discord.com/developers/docs/monetization/entitlements#create-test-entitlement
+ */
+export interface RESTPostAPIEntitlementBody {
+	/**
+	 * ID of the SKU to grant the entitlement to
+	 */
+	sku_id: Snowflake;
+	/**
+	 * ID of the guild or user to grant the entitlement to
+	 */
+	owner_id: Snowflake;
+	/**
+	 * The type of entitlement owner
+	 */
+	owner_type: EntitlementOwnerType;
+}
+
+/**
+ * https://discord.com/developers/docs/monetization/entitlements#create-test-entitlement
+ */
+export type RESTPostAPIEntitlementResult = Partial<Omit<APIEntitlement, 'subscription_id' | 'starts_at' | 'ends_at'>>;
+
+/**
+ * https://discord.com/developers/docs/monetization/entitlements#create-test-entitlement
+ */
+export enum EntitlementOwnerType {
+	Guild = 1,
+	User,
+}
+
+/**
+ * https://discord.com/developers/docs/monetization/entitlements#delete-test-entitlement
+ */
+export type RESTDeleteAPIEntitlementResult = never;
+
+/**
+ * https://discord.com/developers/docs/monetization/skus#list-skus
+ */
+export type RESTGetAPISKUsResult = APISKU[];
```

Comment:
This is not what this returns; look at the JSON body https://discord.com/developers/docs/monetization/skus#list-skus, it includes a lot more fields 👀

Reply:
But they arent documented anywhere

Reply:
Its in the example, ergo documented :mm:
I'd make a PR to document them tho

Reply:
Then i should add this props to APISKU?

Reply:
Well... Check if theres a single get /app/:id/skus/:id route
see if that returns those fields
If the route doesn't exist, add them to api sku
if the route exists and it doesnt return them, it should go in a special type here

Reply:
This is semi-related: https://github.com/discord/discord-api-docs/pull/6477

> The JSON examples **DO** contain fields not included in the table. Fields that can be used by devs **today** are documented in the tables with descriptions. Something we hope to address in the future but for now, rely on the defined fields.

Whilst it wasn't the pull request that added this, there were follow-ups about this.

Reply:
So it will have a followup in the api types, for now it stays undocumented